www/front_src/src/Extensions/Wrapper/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles({
+  contentWrapper: {
+    '@media (min-width: 767px)': {
```

Comment:
It's better to use the breakpoints provided by Material UI
https://mui.com/customization/breakpoints/#css-media-queries